pkg/providers/tinkerbell/tinkerbell.go

```diff
@@ -208,8 +208,14 @@ func (p *Provider) Version(clusterSpec *cluster.Spec) string {
 	return clusterSpec.VersionsBundle.Tinkerbell.Version
 }
 
-func (p *Provider) EnvMap(_ *cluster.Spec) (map[string]string, error) {
-	return map[string]string{}, nil
+func (p *Provider) EnvMap(spec *cluster.Spec) (map[string]string, error) {
+	environment := map[string]string{}
+	if p.datacenterConfig == nil {
+		environment["TINKERBELL_IP"] = p.datacenterConfig.Spec.TinkerbellIP
+	} else {
+		logger.V(2).Info("Tinkerbell datacenter config not specified in provider; cannot define TINKERBELL_IP environment variable")
```

Comment:
should this be an error since if it's not provided, the cluster creation would fail? Or can we just give it a dummy value temporarily?

Reply:
I've just changed to using dummy values for everything because I've realized this configuration is totally superfluous in our context.

See the in-line code commentary on why.